tests/ui/transmutability/references/unit-to-u8.stderr

```diff
@@ -1,8 +1,8 @@
-error[E0277]: `Unit` cannot be safely transmuted into `u8`
+error[E0277]: `&Unit` cannot be safely transmuted into `&u8`
   --> $DIR/unit-to-u8.rs:22:52
    |
 LL |     assert::is_maybe_transmutable::<&'static Unit, &'static u8>();
-   |                                                    ^^^^^^^^^^^ The size of `Unit` is smaller than the size of `u8`
+   |                                                    ^^^^^^^^^^^ The referent size of `&Unit` (0 bytes) is smaller than that of `&Unit` (0) bytes
```

Comment:
This still doesn't look right 😆 

```suggestion
   |                                                    ^^^^^^^^^^^ The referent size of `&Unit` (0 bytes) is smaller than that of `&Unit` (0 bytes)
```

Reply:
Oof, I need to sleep more.

[Done?](https://github.com/rust-lang/rust/compare/107a473cf237396f6251c65210672311c99832ec..422fb2e8421d53fd817d3066be25bf5467394c89#diff-63c8c70fb480c0c10ffa9ba981d76373b6d6b48327cdd7c317b0d75148904dc6R3098)